gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.4-bin.zip
```

Comment:
Should this PR also update `gradle-wrapper.jar` (https://github.com/googleads/google-ads-java/tree/main/gradle/wrapper)?